### DISCUSSION TITLE
Display a Windows message box with Python

### DISCUSSION THREAD

**Discussion by White-Rabbit-Scientific** [beeware/toga#1697] [General]
Blog post, perhaps useful:

**"Display a message box with Python without using a non-standard library or other dependency (Windows)"**

- https://mfcallahan.blog/2022/11/10/display-a-message-box-with-python-without-using-a-library-or-other-dependency-windows/

**Comment by freakboy3742** [comment]
It's not clear why you're sharing this. Toga has a dialogs API which is platform independent; and if you're using Toga, you've already accepted having a dependency in your app.

In addition, the dialog being displayed here is using the win32 API, which results in dialogs which won't exactly match modern apps.
